### DISCUSSION TITLE
Invalid Response form an upstream server when login with password

### DISCUSSION THREAD

**Discussion by ZaidTops** [supabase/supabase#18609] [Questions]
How can we resolve this issue the login with password (signInWithPassword) method of supabase gives error with status 502 invalid response from an upstream server.

**Comment by GaryAustin1** [comment]
More info likely needed as it is pretty generic.
Is this Supabase hosted?  Did this work then stop?  Is the a new instance and if so does anything else work?  Is this the first call after a createClient?  
You could try restarting your instance also.

**Comment by ZaidTops** [comment]
Yes my all of the supabase functions working SSO login also working but for this method I got the error
![Uploading Screenshot 2023-10-30 190851.png…]()

**Reply by GaryAustin1** [reply]
With the info you have given, I have no idea.   Of all the auth calls that is the simplest.  You might try Supabase Discord to see if someone else has ideas if no one else suggests something here.

**Comment by kangmingtay** [comment]
Hi @ZaidTops, an "invalid response from an upstream server" error is usually an indication that the auth service is down and you can reach out to supabase.help for assistance
